### PR TITLE
Gary peters new form

### PR DESCRIPTION
**Pull request by tomdionesotes**
Completely new form. Updated YAML and tested with Yamltron, and it submits successfully: https://s3.amazonaws.com/yamltron-dev/screenshots/error/3ef25ffd-5118-47f6-b7c8-79448f11bf37.png

### CODE REVIEW

File: members/P000595.yaml [hunk 1]
```diff
@@ -1,62 +1,211 @@
...
+          options:
+            Comment on an Issue: opinion
+            Help with a Federal Agency: Help with a Federal Agency
+            Meeting Requests: Meeting Requests
+            Tour Requests: Tour Requests
+            Office Locations: Office Locations
+            Student Resources: Student Resources
+            Veterans Resources: Veterans Resources
+    - wait:
+        - value: 10
```
Comment (kthayer424) [inline members/P000595.yaml]: Can we get this value down further?

Comment (kthayer424) [inline members/P000595.yaml]: Or find a good `find` selector?

Comment (tomdionesotes) [inline members/P000595.yaml]: Are you referring to the amount of time for `wait`?

Comment (kthayer424) [inline members/P000595.yaml]: Yeah. We want to wait the minimum amount of time possible to keep these forms filling out fast.

Comment (tomdionesotes) [inline members/P000595.yaml]: `wait` set at 1 second works. When using `find`, only the `select` form elements have values entered, and the `fill_in` elements are left blank.

**Pull request merged**